examples/minting.py

```diff
@@ -0,0 +1,65 @@
+
+import logging
+from argparse import ArgumentParser
+from typing import cast
+
+from erdpy.accounts import Account, Address
+from erdpy.accounts_repository import AccountsRepository
+from erdpy.proxy import ElrondProxy
+from erdpy.proxy.core import ElrondProxy
+from erdpy.transactions import BunchOfTransactions, Transaction
+
+
+def main():
+    """
+    cd elrond-sdk-erdpy
+    export PYTHONPATH=.
+    python3 ./examples/minting.py \
+        --proxy=https://testnet-gateway.elrond.com \
+        --accounts=./erdpy/testnet/wallets/users \
+        --minter=erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th\
+        --value=1000000000000000000
+    """
+
+    logging.basicConfig(level=logging.ERROR)
+
+    parser = ArgumentParser()
+    parser.add_argument("--proxy", required=True)
+    parser.add_argument("--accounts", required=True)
+    parser.add_argument("--minter", help="minter address", required=True)
+    parser.add_argument("--value", type=int, help="value, as a number (atoms of EGLD)")
+    args = parser.parse_args()
+
+    proxy = ElrondProxy(args.proxy)
+    network = proxy.get_network_config()
+    accounts = AccountsRepository.create_from_folder(args.accounts)
+    minter = cast(Account, accounts.get_account(Address(args.minter)))
```

Comment:
can we rename this file, the variables and so on from `mint` to `airdrop`?

Reply:
Makes sense :rocket: 
Renamed.